client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.jsx

```diff
@@ -137,10 +142,18 @@ const DataFilesToolbar = ({ scheme, api }) => {
     const filteredSelected = selectedFiles.filter(
       (f) => status[f.system + f.path] !== 'SUCCESS'
     );
+
+    const system = selectedFiles[0].system;
+
+    const selectedSystem = systems.find(
+      (s) => s.system === system && s.scheme === scheme
+    );
+
     dispatch({
       type: 'DATA_FILES_TRASH',
       payload: {
         src: filteredSelected,
+        homeDir: selectedSystem?.homeDir || '',
         reloadCallback: reloadPage,
       },
```

Comment:
Not for this PR, but we should implement the same inline-loading behavior for `rename` actions as we have for `trash` actions
![Screen Shot 2023-03-07 at 2 15 51 PM](https://user-images.githubusercontent.com/20326896/223542170-245fa58a-4e8f-478b-a8ed-b4feacabb342.png)

Reply:
task: https://jira.tacc.utexas.edu/browse/FP-2014